ffmpeg.h

```diff
@@ -25,6 +25,12 @@
 
 #endif
 
+/*
+ * AVPixelFormat is a newer release of avutil PixelFormat is the old name
+ */
+#ifndef AVPixelFormat
```

Comment:
I know right?  Only reason you need the backwards compatibility is for Travis and its 12.04 release. There is a way to do 14.04 but seems to have a lot of overhead.  Would that get this to a better place?